src/request/RequestApi.js

```diff
@@ -73,4 +84,39 @@ class RequestApi {
     reject(error) {
         this._reject(error);
     }
+
+    /**
+     * @deprecated Only for database migration
+     */
+    async _checkForMigrationFlag() {
+        const match = document.cookie.match(new RegExp('migrate=([^;]+)'));
+        if (match && match[1] === "1") {
+            await RequestApi.doMigrateAccountsToKeys();
+            document.cookie = 'migrate=0;expires=0'; // Delete the migrate cookie
+        }
+    }
+
+    /**
+     * @deprecated Only for database migration
+     */
+    static async doMigrateAccountsToKeys() {
+        const accountStore = AccountStore.instance;
+        const keyStore = KeyStore.instance;
+
+        const keys = await accountStore.dangerouslistPlain();
+
+        for (const key of keys) {
+            const keyEntry = {
+                encryptedKeyPair: key.encryptedKeyPair,
+                userFriendlyAddress: key.userFriendlyAddress,
+                // Translate between old text type and new number type
+                type: /** @type {EncryptionType} */ (key.type === 'high' ? EncryptionType.HIGH : EncryptionType.LOW)
+            };
+            await keyStore.putPlain(keyEntry);
+        }
+
+        // FIXME Uncomment after/for testing
+        // await accountStore.drop();
```

Comment:
Might want to test that keyStore has been written correctly

Reply:
I trust that it throws an exception if writing fails.